README.md

```diff
@@ -64,6 +52,8 @@ https://sonarqube.com/dashboard/index?id=us.dot.its.jpo.ode%3Ajpo-ode%3Adevelop
 
 ## Getting Started
 
+![Architecture Diagram](images/ODE_Architecture_v2.png)
```

Comment:
Need content around the diagram. Also keep in mind that we will be posting the design document in its entirety in the docs folder. Please commit the document that I sent to Brien before the holidays to the docs folder.